lib/src/command/outdated.dart

```diff
@@ -240,6 +241,11 @@ Consider using the Dart 2.19 sdk to migrate to null safety.''');
         current.version,
         cache,
       );
+
+      final id = current ?? upgradable ?? resolvable ?? latest;
+      final packageAdvisories = await id?.source
+          .getAdvisoriesForPackage(id, cache, Duration(days: 3));
```

Comment:
Nit: You could do
```suggestion
      final packageAdvisories = await id?.source
          .getAdvisoriesForPackage(id, cache, Duration(days: 3)) ?? [];
```
To avoid nullness.

Reply:
done